sdk/web-pubsub/web-pubsub-express/review/web-pubsub-express.api.md

```diff
@@ -62,7 +62,11 @@ export interface DisconnectedRequest {
 export type UserEventRequest = {
     context: ConnectionContext;
     data: string;
-    dataType: "text" | "json";
+    dataType: "text";
+} | {
```

Comment:
I think because we are now widening `data` we have to do a major version bump as this is a breaking change in behavior.

Reply:
This is actually a bug fix....Although in the previous code the definition defines `data` as a `string` when `dataType` is `json`, the value of `data` is actually an `object`....

Reply:
Oh if the current typing is incorrect then I'm fine with fixing it, but can we note it clearly in the changelog?

Reply:
sure